_posts/2019-12-25-technical-debt.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "Technical Debt"
+date:   2019-08-05
+tags: debt devops dora 
+author: Matt Tyler
+image:
+---
+
+> Debt . . . . that peculiar nexus where money, narrative or story, and religious belief intersect, often with explosive force. - Margaret Atwood
+
+Technical debt is an infamous subject in software development. The common understanding is that we accrue it by some series of faustian pacts; short-cuts are taken in order to deliver a feature sooner, with the underlying assumption that additional features will take longer to complete. A development team that continues to accrue technical debt will eventually find time taken to deliver the next feature will begin to approach infinity. In defiance of what is an obvious problem to the average developer, business stakeholders are often loathe to address it. Technical debt is unyielding to measure and containment due to it being a vague concept. If developers wish redress from technical debt, a better way must be found to disclose and appraise the value.
```

Comment:
redress = relief ? 
disclose and appraise the value = describe the cost in real terms.